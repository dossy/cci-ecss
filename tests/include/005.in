@include "x/005.inc";
div { message: $x; }
