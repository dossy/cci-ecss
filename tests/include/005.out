div{message:"goodbye, cruel world!";}
